benefits/pensions.md

```diff
@@ -0,0 +1,38 @@
+# Pensions
+
+We provide a pension plan which everyone is automatically enrolled on.
+
+You will be enrolled as soon as you pass your probation period which will be stated your employment contract.
```

Comment:
Grammar: "...your probation period, which will be stated in your employment contract."